### PR TITLE
zcash_keys: Fix a few problems with no-flags compilation.

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

File: zcash_keys/src/address.rs [hunk 1]
```diff
@@ -143,7 +143,11 @@ impl UnifiedAddress {
 
     /// Returns whether this address has a Sapling receiver.
     pub fn has_sapling(&self) -> bool {
-        self.sapling.is_some()
```
Comment (str4d) [inline zcash_keys/src/address.rs:146]: Document this in a `Fixed` section, as we will be pushing this out in a `0.1.1`.

**Review by str4d — APPROVED** [review]
utACK 39045342df8d236108dce2cb251597d03bb20691 after changelog update.

File: zcash_keys/src/address.rs (hunk 1 above)
Comment (nuttycom) [reply zcash_keys/src/address.rs:146]: Fixed.

**Comment by nuttycom** [comment]
[force-pushed](https://github.com/zcash/librustzcash/compare/39045342df8d236108dce2cb251597d03bb20691..f26ea5d40138e8471e37e3864943cf56e4f82a7e) to note the fix in the CHANGELOG.

**Review by str4d — DISMISSED** [review]
Tested ACK f26ea5d40138e8471e37e3864943cf56e4f82a7e

**Review by str4d — APPROVED** [review]
Re-ACK 836d88b9c305e04e969f0d8f73a3cf25b14928ab

**Review by daira — COMMENTED** [review]
Post-hoc ACK

**Pull request merged**